internal/bminventory/inventory.go

```diff
@@ -2515,16 +2537,40 @@ func (b *bareMetalInventory) UploadHostLogs(ctx context.Context, params installe
 			log.WithError(err).Warnf("Failed to delete temporary files used for upload")
 		}
 	}()
+	if params.LogsType == string(models.LogsTypeHost) {
+		err := b.uploadHostLogs(ctx, params.ClusterID.String(), params.HostID.String(), params.Upfile)
+		if err != nil {
+			return common.GenerateErrorResponder(err)
+		}
+		return installer.NewUploadLogsNoContent()
+	}
+	_, err := b.getCluster(ctx, params.ClusterID.String())
+	if err != nil {
+		common.GenerateErrorResponder(err)
+	}
+	fileName := b.getLogsFullName(params.ClusterID.String(), params.LogsType)
+	log.Debugf("Start upload log file %s to bucket %s", fileName, b.S3Bucket)
+	err = b.objectHandler.UploadStream(ctx, params.Upfile, fileName)
+	if err != nil {
+		log.WithError(err).Errorf("Failed to upload %s to s3", fileName)
+		return common.NewApiError(http.StatusInternalServerError, err)
+	}
```

Comment:
We set nothing on the cluster about these logs (like you do for the host logs...)